diff --git a/Makefile b/Makefile
index 4402ae9..b6ec288 100644
--- a/Makefile
+++ b/Makefile
@@ -107,6 +107,7 @@ build-$(1)-tools-bh:
 	git submodule update --init $$$$reference_riscv_dejagnu riscv-dejagnu; \
 	git submodule update --init $$$$reference_riscv_gcc riscv-gcc; \
 	git submodule update --init $$$$reference_riscv_glibc riscv-glibc; \
+	patch -p0 < ../riscv-sim-hack.diff; \
 	mkdir build; cd build; ../configure --with-arch=$(2) --prefix=/opt/$(1); make
 
 .PHONY: build-$(1)-tools
